runtime/kubernetes/container.go

```diff
@@ -23,6 +23,38 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// These are known kubernetes event Reasons.
+const (
+	// nolint: godot // commented code is not a sentence
+	// known scheduler event reasons can be found here:
+	// https://github.com/kubernetes/kubernetes/blob/v1.23.6/pkg/scheduler/schedule_one.go
+	//reasonFailedScheduling = "FailedScheduling"
+	//reasonScheduled        = "Scheduled"
+
+	// known kubelet event reasons are listed here:
+	// https://github.com/kubernetes/kubernetes/blob/v1.23.6/pkg/kubelet/events/event.go
+
+	// kubelet image event reasons.
```

Comment:
Would it make sense to just use their constants or wrap them at least so we know when they change?

Reply:
Cool, just checking. I just noticed when I went to the code they were public on the package. So, thought they might be importable.

Reply:
Thought I'd provide a bit of clarity on this subject...

The tl;dr is you can vendor from `k8s.io/kubernetes` but its not pretty 😅 

Here's the reason why `go get k8s.io/kubernetes` fails:

https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-505607224

And to actually vendor from `k8s.io/kubernetes`, you have to add a `replace` directive for all nested packages:

https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-505627280

If we want, there are people who've scripted the approach to this so updating the version is easier:

https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-521493597

To see a real world example of what this looks like in the `go.mod` file:

https://github.com/kubernetes-sigs/cloud-provider-huaweicloud/blob/9589bc854c29e399476479f9c5aa9599b2064da5/go.mod#L18-L40

Reply:
Eww... All that work just to reuse some constants... Not worth it imo.

Reply:
Agreed 😄